schemas/subscription-results.v1.schema.json

```diff
@@ -27,17 +27,37 @@
               "properties": {
                 "data": {
                   "type": "array",
-                  "minItems": 0,
+                  "items": {
+                    "type": "object"
+                  }
+                },
+                "meta": {
+                  "type": "array",
                   "items": {
                     "type": "object",
-                    "minProperties": 1,
-                    "additionalProperties": {
-                      "type": ["number", "null"]
-                    }
+                    "properties": {
+                      "name": {
+                        "type": "string"
+                      },
+                      "type": {
+                        "type": "string"
+                      }
+                    },
+                    "additionalProperties": false
                   }
+                },
+                "totals": {
+                  "type": "object"
+                },
+                "profile": {
+                  "type": ["object", "null"]
+                },
+                "trace_output": {
+                  "type": "string"
                 }
               },
-              "required": ["data"]
```

Comment:
Just to clarify, these are extra things that are being sent from snuba? I guess we don't care about them in sentry, just curious why it's different than https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/json_schemas.py